### DISCUSSION TITLE
USBGuard c++ API

### DISCUSSION THREAD

**Discussion by Peteri42** [USBGuard/usbguard#603] [Q&A] [answered]
Hello, i need to make a GUI for USBGuard in c++ and i wanted to ask if it's possible to use your library for communicating with the actual daemon. What i mean exactly is that i'd for example want to create a pointer to DeviceManager inside the daemon so that i could call getDeviceList() and display it inside my own app.

**Comment by radosroka** [comment, marked as answer]
Hello, it's definitely possible. See https://github.com/pinotree/usbguard-applet-qt.

**Reply by Peteri42** [reply]
Okay thank you so much for the answer
